packages/bloc/lib/src/bloc.dart

```diff
@@ -207,46 +206,10 @@ Please make sure to await all asynchronous operations within event handlers.
   Future<void> get future => _completer.future;
 }
 
-/// **@Deprecated - Use `on<Event>` with an `EventTransformer` instead.
-/// Will be removed in v8.0.0**
-///
-/// Signature for a mapper function which takes an [Event] as input
-/// and outputs a [Stream] of [Transition] objects.
-@Deprecated(
-  'Use `on<Event>` with an `EventTransformer` instead. '
-  'Will be removed in v8.0.0',
-)
-typedef TransitionFunction<Event, State> = Stream<Transition<Event, State>>
-    Function(Event);
-
-/// {@template bloc_unhandled_error_exception}
-/// Exception thrown when an unhandled error occurs within a bloc.
-///
-/// _Note: thrown in debug mode only_
-/// {@endtemplate}
-class BlocUnhandledErrorException implements Exception {
-  /// {@macro bloc_unhandled_error_exception}
-  BlocUnhandledErrorException(
-    this.bloc,
-    this.error, [
-    this.stackTrace = StackTrace.empty,
-  ]);
-
-  /// The bloc in which the unhandled error occurred.
-  final BlocBase bloc;
-
-  /// The unhandled [error] object.
-  final Object error;
-
-  /// Stack trace which accompanied the error.
-  /// May be [StackTrace.empty] if no stack trace was provided.
-  final StackTrace stackTrace;
-
-  @override
-  String toString() {
-    return 'Unhandled error $error occurred in $bloc.\n'
-        '$stackTrace';
-  }
+class _Handler {
```

Comment:
Curious: why is this handler made private?

Reply:
It's only used internally so there's no need to expose it publicly.